CONTRIBUTING.md

```diff
@@ -160,6 +160,60 @@ You can also perform the same steps via the [GitHub CLI tool](https://cli.github
 Execute `gh pr create`, and step through the dialog to create the pull request.
 If the branch you will submit against is not the default branch, use the `-B {branch}` option to specify the branch to create the patch against.
 
+#### Performing assertions on the code
+
+We distinguish assertions in two categories:
+
+# Expectations of something that is always supposed to be true
+# Expectations of some input to follow some rule
```

Comment:
```suggestion
- Expectations of something that is always supposed to be true
- Expectations of some input to follow some rule
```

Reply:
@froschdesign is this suggestion driven by personal preferences? I don't mind changing, honestly, it would be good to know why, though 😄

Reply:
@lcobucci
Sorry, I thought it was clear. `#` creates a headline of level 1 but this should be a list, right?

See: https://github.com/laminas/.github/blob/6b4abdb3cef569684fb16fb4259bb3023952591b/CONTRIBUTING.md#performing-assertions-on-the-code

Reply:
Sorry, I'm stupid and mixing up syntaxes (damn you JIRA). I wanted to used `1.` 🤦